pgcli/packages/sqlcompletion.py

```diff
@@ -27,6 +28,15 @@ def suggest_type(full_text, text_before_cursor):
 
     identifier = None
 
+    def strip_named_query(txt):
+        pattern = re.compile(r'\\(n|nd|ns\s+[a-z]+)\s+')
```

Comment:
That's why I used "match" vs "search".
https://docs.python.org/2/library/re.html#search-vs-match